118.md

```diff
@@ -0,0 +1,86 @@
+# NIP-118
+
+## Nostr Double Ratchet Invites
+
+In order to start a [NIP-117](./117.md) Nostr Double Ratchet session, Alice and Bob need to exchange ephemeral public keys and a shared secret.
+
+In a Nostr Double Ratchet invite, Alice gives Bob her
+
+- ephemeral public key, which is also used to respond to the invite on Nostr,
+- shared secret, and
+- Nostr identity public key, which is used to authenticate the exchange.
+
+### Nostr event
+
+Publishing an invite event on Nostr allows other users to start a Double Ratchet session with you.
+
+```typescript
+{
+  pubkey: inviterIdentityKey,
+  content: '',
+  tags: [
+    ['d', `double-ratchet/invites/${uid}`], // uid: unique name for the invitation, e.g. "public"
+    ['l', 'double-ratchet/invites'], // label, for listing all your invites
+    ['ephemeralKey', inviterEphemeralKey],
+    ['sharedSecret', sharedSecret]
+  ],
+  kind: 30078,
+  created_at,
+  id,
+  sig
+}
+```
+
+`l` is a [NIP-32](32.md) label tag that can be used to list your invites. We use this approach with [NIP-78](./78.md) application specific data kind `30078` in order to not bloat the event kind space and instead have a human-readable label.
```

Comment:
If this is for other folks, it should probably not use 30078 and create it's own kind.